### DISCUSSION TITLE
Add menu item on entry editing interface to discard changes

### DISCUSSION THREAD

**Discussion by Saboteur777** [craftcms/cms#8662] [Ideas]
### Description
One of my client accidentally deleted an entry because he misunderstood the meaning of "Delete" in the entry editing screen. I think this can be avoided by introducing a Discard menu option: this will make more clear what Delete means in this context. 

### Steps to reproduce

1. Use Craft :) 
2. Navigate to an entry editing screen. 
3. Click on the caret next to the Save button. 
4. Discard option is missing.

**Comment by brandonkelly** [comment]
If you’re on Craft 3.1 then you can recover the entry by going to the Entries index page, and change the status menu to “Trashed”. Then select the deleted entry and click the “Restore” button.

**Comment by Saboteur777** [comment]
Yes, I know, did exactly this, so we are good right now. This is more like an enhancement suggestion than a bug report, we will be happy if this lands in Craft.
